src/components/tabs/tabs.js

```diff
@@ -574,8 +578,8 @@ export const BTabs = /*#__PURE__*/ Vue.extend({
           controls: safeId ? safeId() : null,
           id: $tab.controlledBy || (safeId ? safeId(`_BV_tab_button_`) : null),
           noKeyNav,
-          posInSet: index + 1,
-          setSize: $tabs.length,
+          posInSet: hidePosinset ? null : index + 1,
```

Comment:
I'd suggest that the camelcase convention is followed - `hidePosInSet` and `hideSetSize`. These follow the camelcase convention that looks to be being used.

Reply:
Thank you, I'v changed the prop names following the camelcase convention.